README.md

```diff
@@ -128,6 +127,7 @@ complete beginners.
 ### 2. Video Courses - Here are 2 from our range :
```

Comment:
Please change `2. Video Courses - Here are 2 from our range :` to `2. Video Courses - Here are 3 from our range :` @shaggyyy2002 . Thank you.